frontend/functions/request.ts

```diff
@@ -109,7 +109,7 @@ export async function requestWithoutAuthentication(
   path: string,
   data?: object
 ): Promise<ResponseType> {
-  const baseUrl = process.env.NEXT_PUBLIC_BACKEND_ORIGIN + '/rorapp/api/';
+  const baseUrl = process.env.NEXT_PUBLIC_BACKEND_ORIGIN + '/api/';
```

Comment:
This is needed:

```suggestion
  const baseUrl = process.env.NEXT_PUBLIC_API_URL;
```

Also, I suggest deleting the `NEXT_PUBLIC_BACKEND_ORIGIN` environment variable from your development environment, since it has been removed in production.

Reply:
I updated it to the newest URL and it works now.